client/signup/steps/plans/index.jsx

```diff
@@ -129,6 +129,7 @@ export class PlansStep extends Component {
 					site={ selectedSite || {} } // `PlanFeaturesMain` expects a default prop of `{}` if no site is provided
 					hideFreePlan={ hideFreePlan }
 					isInSignup={ true }
+					isLaunchPage={ this.props.isLaunchPage }
```

Comment:
`this.props` is already "_destructured_". Not mandatory, but we could add it to the others, e.g.,

```
		const {
			disableBloggerPlanWithNonBlogDomain,
			hideFreePlan,
			isDomainOnly,
                         isLaunchPage,
			selectedSite,
		} = this.props;
```

Reply:
Agree, fixed in [0eb9cb6](https://github.com/Automattic/wp-calypso/pull/30242/commits/0eb9cb68eb1ab7fa50fdaa7a73353596c3f1f88f).